program_119.c

```diff
@@ -0,0 +1,21 @@
+#include <cs50.h>
+#include <stdio.h>
+
+int main(void)
+{
+    int length = get_int("Enter length: ");
+
+    int sumActural = 0;
```

Comment:
I think there is no such word as actural. Please rename to sumActual and totalSum